### PR TITLE
Add show catalogs.

### PR DESCRIPTION
**Pull request by HuangFru**
Optimize some logics.

## Why are the changes needed?

*(For example: The Arctic Flink dataStream API has already supported this configuration: 'scan.startup.mode', but it is not available with Flink SQL. We should expose this configuration to users when they are using Flink SQL.)*

## Brief change log

*(For example:)*
  - *Add the 'scan.startup.mode' configuration to the ArcticScanContext in the flink modules*
  - *The flink 1.12 and 1.14 versions are affected.*
  - *Add the documentation in the flink chapter*

## How was this patch tested?
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] Run test locally before making a pull request

## Documentation

  - Does this pull request introduces a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: ams/ams-server/src/main/java/com/netease/arctic/ams/server/repair/command/ShowCall.java [hunk 1]
```diff
@@ -1,27 +1,30 @@
...
-    this.arcticCatalog = arcticCatalog;
+  public ShowCall(CatalogManager catalogManager, Namespaces namespaces) {
+    this.catalogManager = catalogManager;
     this.namespaces = namespaces;
   }
 
   @Override
-  public String call(Context context) {
+  public String call(Context context) throws FullTableNameException {
+    if (context.getCatalog() == null) {
```
Comment (shidayang) [inline ams/ams-server/src/main/java/com/netease/arctic/ams/server/repair/command/ShowCall.java:25]: show catalog support context.getCatalog() is null

**Review by shidayang — CHANGES_REQUESTED** [review]

**Review by shidayang — APPROVED** [review]
